tests/test.rfb.js

```diff
@@ -768,6 +768,26 @@ describe('Remote Frame Buffer protocol client', function () {
                                                                                     13, 9, 0x0);;
             });
 
+            it('should send button messages when longpress with small movement', function () {
```

Comment:
Is this a changed behaviour? Otherwise, I think they should also be earlier to indicate you are keeping existing behaviour.